jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/format/ProcessorSection.java

```diff
@@ -80,9 +80,9 @@ protected void summary(BufferedWriter out) throws IOException
          * These String arrays refer to enum values in entries of the
          * ProcessorInfo structure defined in xhpi.h .
          */
-        String[] Archs      = { "Unknown", "x86", "S390", "Power", "IA64", "S390X", "AMD64", "RISCV"};
+        String[] Archs      = { "Unknown", "x86", "S390", "Power", "IA64", "S390X", "AMD64", "RISCV", "AArch64" };
         String[] SubTypes   = { "i486", "i586", "Pentium II", "Pentium III",
-            "Merced","McKinley", "PowerRS", "PowerPC", "GigaProcessor", "ESA", "Pentium IV", "T-Rex", "Opteron", "RV64G"};
+            "Merced","McKinley", "PowerRS", "PowerPC", "GigaProcessor", "ESA", "Pentium IV", "T-Rex", "Opteron", "RV64G", "Armv8-A" };
```

Comment:
Please indent this like line 87, add the missing space after the first comma and fold it so it's no longer than line 83.

Reply:
Changed the indentation and line folding.